README.md

```diff
@@ -366,7 +357,18 @@ $ cat /sys/module/zfs/parameters/zfs_dmu_offset_next_sync
 
 More details can be found [in the OpenZFS documentation](https://openzfs.github.io/openzfs-docs/Performance%20and%20Tuning/Module%20Parameters.html).
 
-# Credits
+## Project and maintainer
+
+The bmap-tools project implements bmap-related tools and API modules. The
+entire project is written in python and supports python 2.7 and python 3.x.
```

Comment:
Just a quick note: I think we should drop 2.x and support only 3.6 and above....